server/middleware/validation.js

```diff
@@ -26,6 +26,7 @@ const validationErrors = {
     isAfterMinYear: label => `${label} must be after ${MIN_ALLOWABLE_YEAR}`,
     isValidDay: label => `${label} must contain a real day`,
     isValidWithinGivenDays: (label, props) => `${label} must not be more than ${props} days in the future`,
+    isValidWithinPastGivenDays: (label, props) => `${label} must not be more than ${props} days in the past`,
```

Comment:
```suggestion
    isValidWithinPastGivenDays: (label, props) => `${label} must be within the last ${props} days.`,
```
I'm not overly fussed, but this feels less wordy.

Reply:
Might be worth looking at the following: https://design-system.service.gov.uk/components/error-message/